auth/src/main/res/layout/fui_check_email_layout.xml

```diff
@@ -7,31 +7,44 @@
     android:layout_height="match_parent">
 
     <LinearLayout
```

Comment:
I can't motivate you to use ConstraintLayout? 😉

Reply:
I think once @lsirac finishes the ToS update I'll have to use `ConstraintLayout` to get around the merge conflicts anyway.